src/index.js

```diff
@@ -100,14 +146,12 @@ function initDataLayer(instanceName) {
  * @param {String} instanceName the name of the data layer instance in the global scope
  * @param {Array<String>} tags the GA4 tags to initialize
  */
-function initGa4(instanceName, tags) {
+function initGa(instanceName, tags) {
   // eslint-disable-next-line no-console
   console.assert(gtm.config.analytics, 'Analytics is disabled in the martech config');
+  const attrs = { async: true };
```

Comment:
I'd even move this as the default `attrs` in `loadScript` so you don't have to repeat it

Reply:
Done